packages/server/lib/cloud/protocol.ts

```diff
@@ -277,11 +305,7 @@ export class ProtocolManager implements ProtocolManagerShape {
     }
   }
 
-  async uploadCaptureArtifact ({ uploadUrl, fileSize, filePath }: CaptureArtifact): Promise<{
-    success: boolean
-    fileSize: number | bigint
-    specAccess: ReturnType<AppCaptureProtocolInterface['getDbMetadata']>
-  } | undefined> {
+  async uploadCaptureArtifact ({ uploadUrl, fileSize, filePath }: CaptureArtifact, captureErrorsOverride?: boolean) {
```

Comment:
Is captureErrorsOverride just used for testing purposes?

Reply:
Yep. Could refactor a bit to have caller pass in the env var to make things a little cleaner.